chains/orchestrator-paras/runtime/dancebox/src/lib.rs

```diff
@@ -1729,6 +1737,27 @@ impl pallet_multisig::Config for Runtime {
     type WeightInfo = weights::pallet_multisig::SubstrateWeight<Runtime>;
 }
 
+pub struct MockCurrentSessionGetter;
+
+impl tp_traits::GetSessionIndex<u32> for MockCurrentSessionGetter {
+    fn session_index() -> u32 {
+        1
+    }
+}
+
+impl pallet_inactivity_tracking::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type CollatorId = CollatorId;
+    type MaxInactiveSessions = ConstU32<5>;
+    type MaxCollatorsPerSession = ConstU32<100>;
+    #[cfg(not(feature = "runtime-benchmarks"))]
+    type CurrentSessionIndex = CurrentSessionIndexGetter;
+    #[cfg(feature = "runtime-benchmarks")]
+    type CurrentSessionIndex = MockCurrentSessionGetter;
```

Comment:
this should not be needed for benchmarks I think. Why did we need a mockCurrentSessionGetter?

Reply:
in theory, everything should work without the mocker.

Reply:
Removed the mocker in https://github.com/moondance-labs/tanssi/pull/922/commits/2d880c174d44e0c3a0fab082e79e73d7bda818fe